softlayer/resource_softlayer_scale_group.go

```diff
@@ -245,9 +245,8 @@ func buildScaleVlansFromResourceData(d *schema.Set, meta interface{}) ([]datatyp
 
 		if len(networkVlans) < 1 {
 			return nil, fmt.Errorf(
-				"Unable to locate a vlan matching the provided router hostname and vlan number: %s/%s",
-				primaryRouterHostname,
-				vlanNumber)
+				"Unable to locate the specified vlan: %d.%s",
+				vlanNumber, primaryRouterHostname)
```

Comment:
> It may be more recognizable to us, but there is no such thing as vlan '1234.bcr01a.sjc03'. It isn't a thing, and a user new to SoftLayer will find this confusing.

You are right. I just got the order wrong :-)
![screen_shot_2016-10-03_at_10_23_21_pm](https://cloud.githubusercontent.com/assets/139994/19060718/77bf6ebc-89b8-11e6-9ca3-92b46afc6d60.png)